lib/rack/attack/store_proxy/redis_cache_store_proxy.rb

```diff
@@ -10,17 +10,19 @@ def self.handle?(store)
           store.class.name == "ActiveSupport::Cache::RedisCacheStore"
         end
 
-        def increment(name, amount = 1, **options)
-          # RedisCacheStore#increment ignores options[:expires_in].
-          #
-          # So in order to workaround this we use RedisCacheStore#write (which sets expiration) to initialize
-          # the counter. After that we continue using the original RedisCacheStore#increment.
-          if options[:expires_in] && !read(name)
-            write(name, amount, options)
+        if defined?(ActiveSupport) && ActiveSupport::VERSION::MAJOR < 6
```

Comment:
I'd suggest adding a gemfile that runs `spec/acceptance/stores/active_support_redis_cache_store_spec.rb` with activesupport < 6

Reply:
Separate PR?

Reply:
Done https://github.com/rack/rack-attack/pull/632